src/main.py

```diff
@@ -76,15 +70,10 @@ def is_yaml_file(f: str) -> bool:
     return f.endswith(".yml") or f.endswith(".yaml")
 
 
-async def handle_event(event: Dict, event_context, projects_ids: Optional[List[str]] = None, services: Optional[List[GCPService]] = None):
-    if isinstance(event_context, Dict):
-        # for k8s installation
-        context = LoggingContext(event_context.get("execution_id", None))
-    else:
-        context = LoggingContext(None)
-
+async def query_metrics(execution_id: Optional[str], services: Optional[List[GCPService]] = None):
+    context = LoggingContext(execution_id)
     if not services:
-        # load services for GCP Function
+        # Load services for GCP Function and for tests
```

Comment:
why do we add this "and for tests", what does it mean??

Reply:
Tests call the function without the services param. Nevertheless, this is one of the things to change, probably.